pom.xml

```
@@ -23,8 +23,7 @@
 
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <java.level>8</java.level>
-    <jenkins.version>2.303.3</jenkins.version>
+    <jenkins.version>2.375.2</jenkins.version>
```

The ["Choosing a Jenkins version" page](https://www.jenkins.io/doc/developer/plugin-development/choosing-jenkins-baseline/) recommends 2.361.4 when making the transition to Java 11 and plugin bill of materialsl
```suggestion
    <jenkins.version>2.361.4</jenkins.version>
```